zabbix/hooks/run

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+
+exec 2>&1
+#start zabbix server
+{{pkg.path}}/sbin/zabbix_server -c {{pkg.svc_config_path}}/zabbix_server.conf
+{{pkg.path}}/sbin/zabbix_agentd -c {{pkg.svc_config_path}}/zabbix_agentd.conf
+
+#habitat requires a hang when it is ending so just made a loop to hang
```

Comment:
It's technically not a hang that habitat needs. This has more to how supervisors work where it needs to lock into the process for management. 

You most likely want zabbix server or zabbix agent to be managed by hab-sup and not run any of these processes in dameon mode.